bulk-export/src/main/java/no/sikt/nva/data/report/api/export/CsvTransformer.java

```diff
@@ -31,6 +31,7 @@
 
 public class CsvTransformer extends BulkTransformerHandler {
 
+    private static final String CONTENT_TYPE_TEXT_CSV = "text/csv";
```

Comment:
Nitpick: Since this class only handles CSV, could the constant simply be named "CONTENT_TYPE"?

It could in theory include other metadata as well, such as encoding (UTF-8 for example)?